pkg/machinery/constants/constants.go

```diff
@@ -992,6 +992,9 @@ const (
 
 	// SyslogListenSocketPath is the path to the syslog socket.
 	SyslogListenSocketPath = "/dev/log"
+
+	// MinimumGOAMD64Level is the minimum x86_64 microarchitecture level required by Talos.
+	MinimumGOAMD64Level = 2
```

Comment:
Hello @smira, the doc said, I can use `GOAMD64=v1` to build talos for the old CPU arch. But look like this const never set to `1` base on the `GOAMD64` value.

https://github.com/siderolabs/talos/blob/bc8bf9e8a865473011dcf5ad2df08c8ff1af5110/website/content/v1.7/advanced/building-images.md?plain=1#L47-L52

Reply:
yes, you would need to change this field as well.

Reply:
Thank you,

`GOAMD64=v1 GO_LDFLAGS="-X github.com/siderolabs/talos/pkg/machinery/constants.MinimumGOAMD64Level=1"` should work, right?

Reply:
I don't have a good way to test this, but I would assume so. We almost never had a user who actually had amd64-v1 device, but it was always misconfiguration of the VM assuming lower level of amd64.

Reply:
I just want to use an old laptop to run Talos, its CPU was old, but still able to handle the workload for home lab. :)

Reply:
I can confirm it works. However, I have to change the sourcecode to update `MinimumGOAMD64Level` to 1. It can't be change via `-X` (only work for variable, not constants).

Is there any change you can change it to a variable instead, I don't want to modify the source code. Just want build new version using build args.